src/pdf.sandbox.external.js

```diff
@@ -81,6 +81,13 @@ class SandboxSupportBase {
         ) {
           return;
         }
+
+        if (callbackId === 0) {
+          // This callbackId corresponds to the one used for userActivation.
+          // So here, we cancel the last userActivation.
+          this.win.clearTimeout(this.timeoutIds.get(0));
```

Comment:
Let's avoid the additional hard-coding here:
```suggestion
          this.win.clearTimeout(this.timeoutIds.get(callbackId));
```